pipeline_steps/ssh_slave.groovy

```diff
@@ -6,61 +6,51 @@
  *  - playbooks/inventory/hosts
  */
 def connect(){
-  common.conditionalStage(
```

Comment:
This change seems a bit out of place here.  If we're going to nuke these stages/steps then we probably need to clean up Connect Slave / Destroy Slave from STAGES.